src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets

```diff
@@ -209,17 +219,59 @@ Copyright (c) .NET Foundation. All rights reserved.
        * Assets from the referenced packages. These will be implicitly included when nuget
          restores the package and includes the package props file for the package.
   -->
+
+    <!-- StaticWebAssets from the current project come from ResolveCurrentProjectStaticWebAssetsInputs which is a dependency of this target. -->
+    <!-- StaticWebAssets from referenced projects come from ResolveReferencedProjectsStaticWebAssets which is a dependency of this target. -->
+    <!-- StaticWebAssets from packages are already available, so we don't do anything. -->
   <Target
     Name="ResolveStaticWebAssetsInputs"
-    DependsOnTargets="$(ResolveStaticWebAssetsInputsDependsOn)">
+    DependsOnTargets="$(ResolveStaticWebAssetsInputsDependsOn)" />
+
+  <!-- This is a helper task to compute the project references we need to invoke to retrieve
+       the static assets for a given application. We do it this way so that we can
+       pass additional build properties to compute the assets from the package when referenced
+       as a project. For example, Identity uses this hook to extend the project reference and
+       pass in the bootstrap version to use.
+  -->
 
-    <!-- StaticWebAssets from the current project -->
+  <Target Name="ResolveReferencedProjectsStaticWebAssets" DependsOnTargets="$(ResolveReferencedProjectsStaticWebAssetsDependsOn)">
 
-    <PropertyGroup>
-      <StaticWebAssetBasePath Condition="$(StaticWebAssetBasePath) == ''">_content/$(PackageId)</StaticWebAssetBasePath>
-    </PropertyGroup>
+    <ItemGroup>
+      <_StaticWebAssetsProjectReference Include="@(AnnotatedProjects)" />
```

Comment:
Despite the underscore, @(_MSBuildProjectReferenceExistent) seems like the right thing over @(AnnotatedProjects).

Reply:
I didn't want to depend on _MSBuildProjectReferenceExistent as it's prefixed by underscore and common convention means "private" is that not the case? Using that property is my preference, but want to make sure it's ok to take a dependency on that.

Reply:
My quick reading is that there are cases where things are in _MSBuildProjectReferenceExistent, but not AnnotatedProjects. When SkipGetTargetFrameworkProperties=true.

I agree about the underscore, but I'm not actually finding what else you should use here. @rainersigwald, what's your take?

Reply:
You're right to be concerned, but `_MSBuildProjectReferenceExistent` is "public". You wouldn't be the first or the 20th to take a dependency on it. Using that is a better choice here.

Reply:
Switched to use _MSBuildProjectReferenceExistent